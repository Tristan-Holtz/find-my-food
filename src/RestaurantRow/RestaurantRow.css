.restaurant-row {
  background-color: #fff;

  border-top: 1px solid orange;
  box-shadow: 0 4.5px 0 0 orange;
  display: flex;
  justify-content: space-between;
  margin: 1em;
  padding: 1em;
  -webkit-transition: box-shadow 0.25s ease-out;
}

.restaurant-row:hover {
  box-shadow: 0 7.5px 0 0 darkorange;
}

.row_div-right {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
}

.row_div-left {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
}

h2 {
  font-style: italic;
}
