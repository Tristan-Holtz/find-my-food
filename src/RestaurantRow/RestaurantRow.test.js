import React from "react";
import RestaurantRow from "./RestaurantRow";
import { shallow } from "enzyme";

describe("RestaurantRow", () => {
  let props = {
    genre: "Asian",
    name: "Uchi",
    city: "Denver",
    state: "CO",
    telephone: "(000)123-4567",
    address1: "Somewhere in RiNo, I forget",
    id: "98whef982eifhw",
  };
  let wrapper = shallow(<RestaurantRow info={props} />);
  it("should render the component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
