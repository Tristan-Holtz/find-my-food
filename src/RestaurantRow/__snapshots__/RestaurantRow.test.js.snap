// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantRow should render the component 1`] = `ShallowWrapper {}`;
